netdisco/discoverables/lg_soundbar.py

```diff
@@ -0,0 +1,10 @@
+"""Discover LG smart devices."""
+from . import MDNSDiscoverable
+
+
+# pylint: disable=too-few-public-methods
+class Discoverable(MDNSDiscoverable):
+    """Add support for discovering LG smart devices."""
+
+    def __init__(self, nd):
+        super(Discoverable, self).__init__(nd, '_lg-smart-device._tcp.local.')
```

Comment:
The domain name does not look like it's limited to soundbars. We should rename this file to lg_smart_device.py

Reply:
It's a generic name, but I haven't been able to find anything other than their soundbars that responds this way. Happy to make the change, though.

Reply:
I think it's better to keep it generic so we don't have to do a breaking change if LG starts sharing code between teams.